config/src/loaders/toml.rs

```diff
@@ -88,18 +90,18 @@ mod tests {
         let filename = Path::new("config.toml");
         let config = super::from_file(&filename).unwrap();
 
-        assert_eq!(config, Config::default());
+        assert_eq!(config, PartialConfig::default());
     }
 
     #[test]
     fn test_load_config_from_file() {
         super::FILE_CONTENTS.with(|cell| {
             cell.set(
                 r"
-environment = 'testnet-1'
-[connections]
-inbound_limit = 999
-",
+    environment = 'testnet-1'
```

Comment:
Remove extra whitespace
```suggestion
environment = 'testnet-1'
```